expense-tracker/src/db/ddl/create_scripts.sql

```diff
@@ -20,7 +21,8 @@ create table t_wallet_details(
   f_wallet_status varchar(20),
   f_created_by bigint REFERENCES t_user_details(f_id),
   f_created_time timestamp default current_timestamp,
-  f_updated_time timestamp default current_timestamp
+  f_updated_time timestamp default current_timestamp,
+  UNIQUE (f_wallet_name, f_created_by)
```

Comment:
Why created by has to be unique ? Isn't it good enough if wallet names are unique ?